core/src/main/java/org/apache/calcite/model/ModelHandler.java

```diff
@@ -173,6 +176,18 @@ public static void addFunctions(SchemaPlus schema, String functionName,
         + "'initAdd', 'merge' and 'result' methods.");
   }
 
+  private ObjectMapper getJsonObjectMapper() {
```

Comment:
What do you think of caching JSON_MAPPER and YAML_MAPPER instances to the final static fields?

`ObjectMapper` is thread safe, and it should be fine to just reuse it

Reply:
Thank you. Agree.
I guess the same could be applied to Json/YamlModelTest